### ISSUE TITLE
web user interface accessibility issues for totally blind college students studying audio engineering

### ISSUE THREAD

**Issue by nickgaw** [Eddycrack864/UVR5-UI#66]
I myself am a totally blind student studying audio engineering and when I try to use this tool for separating music into stems I find that lots of the interface is not very usable with the keyboard.  First, I am unable to using any screen reader like the free NVDA screen reader to tab threw and select models after running the installation batch file under windows 11 pro with all updates installed.  If I try to select an audio file like a mp3 file I am able to play the file in the interface and change the speed of it but when I try to separate the file into stems I get an error message in the web interface but no clue of what it is as it is not displayed inside of the user interface.  If I try to press any of the model buttons like all models or installed models nothing happens from a screen reader's point of view and with accessibility standards properly put in then status messages and other informational messages would be spoken.  http://w3.org/WAI and http://www.webaim.org has great web page and user interface accessibility information.  For windows systems http://www.microsoft.com/enable also has very useful information and the linux accessibility pages like the orca screen reader also has very good information.  Could you please look into making the interface more accessible for keyboard only users using screen readers as all platforms both desktop and mobile have some screen reading functionality built in and it would be nice for us students to be able to use this software?
